assets/app/scripts/controllers/build.js

```diff
@@ -70,6 +71,15 @@ angular.module('openshiftConsole')
               }
               $scope.build = build;
             }));
+            watches.push(DataService.watchObject("buildconfigs", $routeParams.buildconfig, context, function(buildConfig, action) {
+              if (action === "DELETED") {
+                $scope.alerts["deleted"] = {
+                  type: "warning",
+                  message: "This build configuration has been deleted."
```

Comment:
This message seems to imply what you're looking at has been deleted. Maybe something like "Build configuration 'my-build-config' has been deleted."?